src/electionguard/encode.py

```diff
@@ -0,0 +1,50 @@
+"""Converting module that define padding and default settings for byte order and byte encoding."""
+
+from enum import IntEnum
+
+
+BYTE_ORDER = "big"
+BYTE_ENCODING = "utf-8"
+
+PAD_INDICATOR_SIZE = 2
+_PAD_BYTE = b"\x00"
+
+
+class PaddedDataSize(IntEnum):
+    """Define the sizes for padded data."""
+
+    Bytes_32 = 32 - PAD_INDICATOR_SIZE
```

Comment:
This was the most creative way I could come up with to showcase the relationship of the padding to the padding indicator size. This may be an argument where it more clever than understandable.

Reply:
These values would show up in the json as 30, 62, etc. as one would expect.